lib/stdlib/doc/src/gen_server.xml

```diff
@@ -798,6 +871,10 @@ gen_server:abcast     -----> Module:handle_cast/2
         <v>Status = term()</v>
       </type>
       <desc>
+        <warning>
+          <p>This callback is deprecated, in new code use <seemfa marker="#Module:format_status/1">
+          format_status/1</seemfa>.</p>
```

Comment:
Add a note that the this old callback function will not be called if the new one is present.